### DISCUSSION TITLE
Clone Builder Blocks

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#3658] [Ideas]
Much like the Repeater class has a `cloneable()` method to allow cloning repeater items, it would be super helpful if the Builder class had a `cloneable()` method to allow cloning Builder blocks.

**Comment by danharrin** [comment]
Added to the roadmap

**Comment by petarcar** [comment]
@danharrin At the moment if block has `maxItems(1)` one can still clone it. If this is not on purpose, it would be nice for `maxItems()` on the block to take over if builder is `cloneable()`. Thanks a bunch! Cheers.

**Reply by danharrin** [reply]
Thanks for letting me know, but please open an issue with a reproduction repository so we can track the bug

**Reply by petarcar** [reply]
Does it have to be repo, or I can post an reproduction example in the issue?

**Reply by petarcar** [reply]
BTW ... WOW for reply time! ❤️

**Reply by petarcar** [reply]
Ah got it! Clear.
